src/allocators.zig

```diff
@@ -229,6 +229,11 @@ pub fn BSSList(comptime ValueType: type, comptime _count: anytype) type {
                 this.data[index] = item;
                 return &this.data[index];
             }
+
+            pub fn deinit(this: *OverflowBlock) void {
+                if (this.prev) |p| p.deinit();
+                bun.default_allocator.destroy(this);
```

Comment:
`self.allocator.create` is used to allocate `OverflowBlock`. Can we ensure it's `bun.default_allocator`?

Reply:
https://github.com/oven-sh/bun/blob/nektro-patch-42406/src/fs.zig#L100-L101

Reply:
https://github.com/oven-sh/bun/blob/300f321dec99830d5a0c73e56a01ae2bce99d400/src/bun.zig#L140

Reply:
alias removed in https://github.com/oven-sh/bun/pull/22782